apps/omg_status/lib/status.ex

```diff
@@ -23,7 +23,7 @@ defmodule OMG.Status do
   # if ETS table has a tuple entry in form of {:boot_in_progress, 0}, return false
   # if ETS table has a tuple entry in form of {:boot_in_progress, 1}, return true
   @health_match List.flatten(
-                  for n <- [:boot_in_progress, :ethereum_client_connection, :main_supervisor_halted],
+                  for n <- [:boot_in_progress, :ethereum_connection_error, :main_supervisor_halted],
```

Comment:
should we add `ethereum_stalled_sync` ? they both mean that the service isn't operational...